torch/distributed/distributed_c10d.py

```diff
@@ -368,7 +368,20 @@ def init_process_group(backend,
             the process group. Default value equals 30 minutes.
             This is applicable for the ``gloo`` backend. For ``nccl``, this is
             applicable only if the environment variable ``NCCL_BLOCKING_WAIT``
-            is set to 1.
+            or ``NCCL_ASYNC_ERROR_HANDLING`` is set to 1. When
+            ``NCCL_BLOCKING_WAIT`` is set, this is the duration for which the
+            process will block and wait for collectives to complete before
+            throwing an exception. When ``NCCL_ASYNC_ERROR_HANDLING`` is set,
+            this is the duration after which collectives will be aborted
+            asynchronously and the process will crash. ``NCCL_BLOCKING_WAIT``
+            will provide errors to the user which can be caught and handled,
+            but due to its blocking nature, it has a performance overhead. On
+            the other hand, ``NCCL_ASYNC_ERROR_HANDLING`` has little
```

Comment:
There should be no perf overhead with `NCCL_ASYNC_ERROR_HANDLING` right? If so, should we update the comment to mention that. If not, let's just replace `little` with `very little` to emphasis not having a significant overhead.

Reply:
It's not 0 perf overhead since there is a slight QPS regression measured, and it slowly closes the gap in each iteration, so I didn't want to be too absolute and say "no perf overhead".